addon/session-stores/cookie.js

```diff
@@ -120,7 +120,11 @@ export default BaseStore.extend({
   */
   _cookieExpirationTime: null,
   cookieExpirationTime: persistingProperty(function(key, value) {
-    if (value < 90) {
+    // When nulling expiry time on purpose, we need to clear the cached value.
+    // Otherwise, `_calculateExpirationTime` will reuse it.
+    if (value === null) {
+      this.get('_cookies').clear(`${this.get('cookieName')}-expiration_time`);
```

Comment:
I don't know why this is causing
```
  1) Graceful regression tests does not error when using the adaptive store:
     AssertionError: expected undefined to deeply equal [ Array(1) ]
```
which asserts 
```
expect(response.headers['set-cookie']).to.deep.equal(['ember_simple_auth-session=%7B%22authenticated%22%3A%7B%7D%7D; path=/'])
```

Reply:
Try reproducing this outside of the tests - run `ember s` and have a look at the response.

Reply:
Also did you check what the header value is actually (add `console.log(response.headers['set-cookie'])` in the line before the assertion)?